src/client.js

```diff
@@ -646,6 +646,12 @@ class Client {
         if (instanceKey.startsWith("http://")) instanceKey = instanceKey.substr(7);
         if (instanceKey.startsWith("https://")) instanceKey = instanceKey.substr(8);
         const rootKey = `acc.js.sdk.${sdk.getSDKVersion().version}.${instanceKey}.cache`;
+        // Clear storage cache if the sdk versions or the instances are different
+        for (let key in this._storage) {
+          if (key.startsWith("acc.js.sdk.") && !key.startsWith(rootKey))
+            this._storage.removeItem(key);
+            console.log(`Cache with the key '${key}' has been removed`);
```

Comment:
There's no logging mechanism (yet) in the SDK. Can you remove this console.log statement?

Reply:
done